bot/utils/moderation.py

```diff
@@ -14,16 +14,17 @@
 
 
 async def post_infraction(
-    ctx: Context, user: Union[Member, Object, User],
-    type: str, reason: str, expires_at: datetime = None, hidden: bool = False
+    ctx: Context, user: Union[Member, Object, User], type: str, reason: str,
+    expires_at: datetime = None, hidden: bool = False, active: bool = True
```

Comment:
I think you agreed to do one-per-line for some other function so this is a candidate for it too.

Reply:
Done.